src/screens/MainApp/DiagnoseRequest/screens/Gallery/index.js

```diff
@@ -0,0 +1,197 @@
+import React, { useState, useEffect, useCallback, useMemo } from 'react'
+import CameraRoll from '@react-native-community/cameraroll'
+import { FlatList, View, Alert, PermissionsAndroid } from 'react-native'
+import { SvgXml } from 'react-native-svg'
+import Background from '~/components/Background'
+import { PrimaryButton, Description, Button, Body } from '~/components'
+import CameraLogo from '~/assets/images/Gallery/camera.svg'
+import * as ImageService from '~/imageService'
+import ListItem from './components/ListItem'
+import styles, { SCREEN_WIDTH } from './styles'
+
+const PAGE_SIZE = 10
+const MAX_SELECTION_COUNT = 10
+const MIN_SELECTION_COUNT = 3
+const DEFAULT_COLUMNS_COUNT = 3
+
+const getPhotos = (endCursor) => {
+  const config = {
+    first: PAGE_SIZE,
+    assetType: 'Photos'
+  }
+
+  if (endCursor) config.after = endCursor
+
+  return CameraRoll.getPhotos(config)
+}
+
+const alertPermissionsError = () => {
+  Alert.alert('Error', 'No se pudo acceder al almacenamiento del dispositivo. Verifique los permisos de la aplicación.')
+}
+
+const requestPermissions = async (permission) => {
+  return PermissionsAndroid.request(permission)
+}
+
+const requestReadPermissions = async () => {
+  return requestPermissions(PermissionsAndroid.PERMISSIONS.READ_EXTERNAL_STORAGE)
+}
+
+const requestWritePermissions = async () => {
+  return requestPermissions(PermissionsAndroid.PERMISSIONS.WRITE_EXTERNAL_STORAGE)
+}
```

Comment:
Please consider moving this to a separate folder like `permissions` or `permissionService`